### ISSUE TITLE
unable to download the singularity image

### ISSUE THREAD

**Issue by flegeai** [MetazoaPhylogenomicsLab/FANTASIA#9]
Hi, 
I'd like to test the FANTASIA program on some proteome sets. But, I am unable to download the .sif. 
When I run the following command :+1: 
 ```
singularity pull --arch amd64 library://gemma.martinezredondo/fantasia/
fantasia:sha256.299ed9bb0947d19d58068f1ecef0128b147bbce73f762570b8ba44141d7b5fe6
```
I got the following error : 
```
FATAL:   Unable to get library client configuration: remote has no library client (see https://apptainer.org/docs/user/latest/endpoint.html#no-default-remote)
```
Would you please check?
Best wishes!

**Comment by jacgonisa** [comment]
> Hi, I'd like to test the FANTASIA program on some proteome sets. But, I am unable to download the .sif. When I run the following command 👍
> 
> ```
> singularity pull --arch amd64 library://gemma.martinezredondo/fantasia/
> fantasia:sha256.299ed9bb0947d19d58068f1ecef0128b147bbce73f762570b8ba44141d7b5fe6
> ```
> 
> I got the following error :
> 
> ```
> FATAL:   Unable to get library client configuration: remote has no library client (see https://apptainer.org/docs/user/latest/endpoint.html#no-default-remote)
> ```
> 
> Would you please check? Best wishes!

I was also running into this problem and it's just a matter of following the documentation https://apptainer.org/docs/user/latest/endpoint.html#no-default-remote

You have to manually set up the library protocol with
```

$ apptainer remote add --no-login SylabsCloud cloud.sycloud.io
INFO:    Remote "SylabsCloud" added.
$ apptainer remote use SylabsCloud
INFO:    Remote "SylabsCloud" now in use.
```

I think I am still having a problem though when I try to download the image

```
WARNING: failed to get key material: 404 Not Found: entity not found

WARNING: integrity: signature object 5 not valid: openpgp: signature made by unknown entity

WARNING: Skipping container verification

```

**Comment by gmartinezredondo** [comment]
Hi!

@jacgonisa are you able to download the image when you change the library protocol even if you get those warnings?

Best,

Gemma

**Comment by jacgonisa** [comment]
> Hi!
> 
> @jacgonisa are you able to download the image when you change the library protocol even if you get those warnings?
> 
> Best,
> 
> Gemma

Yes! the download is successful... However I could not run it with singularity because some paths were hardcoded I think and referred to your paths in your local computer I guess... But I was able to run FANTASIA with the manual installation!

**Comment by gmartinezredondo** [comment]
> But I was able to run FANTASIA with the manual installation!

Perfect! It's great that you could run it :)

And thanks for the help with the download issue.

> However I could not run it with singularity because some paths were hardcoded I think and referred to your paths in your local computer I guess

I'm aware of some issues with the paths. It fails in some systems unless you run it from the home directory. We're working on a newer version that won't have these path issues
